cc/imgproc/imgprocBindings.h

```diff
@@ -204,6 +204,95 @@ namespace ImgprocBindings {
 		  };
 	  };
   };
+
+	class Accumulate : public CvClassMethodBinding<Mat> {
+	public:
+		cv::Mat dst;
+		
+		void createBinding(std::shared_ptr<FF::Value<cv::Mat>> self) {
+			auto src = req<Mat::Converter>();
+			auto mask = opt<Mat::Converter>("mask", cv::noArray().getMat());
+			executeBinding = [=]() {
+				auto depth = self->ref().depth();
+				if (depth != CV_32F && depth != CV_64F)
+					throw std::runtime_error("dst must have a depth of CV_32F or CV_64F - mat depth:" + std::to_string(depth));
+				cv::accumulate(src->ref(), self->ref(), mask->ref());
+				dst = self->ref();
+			};
+		};
+
+		v8::Local<v8::Value> getReturnValue() {
+			return Mat::Converter::wrap(dst);
+		};
+	};
+
+	class AccumulateProduct : public CvClassMethodBinding<Mat> {
+	public:
+		cv::Mat dst;
+
+		void createBinding(std::shared_ptr<FF::Value<cv::Mat>> self) {
+			auto src1 = req<Mat::Converter>();
+			auto src2 = req<Mat::Converter>();
+			auto mask = opt<Mat::Converter>("mask", cv::noArray().getMat());
+
+			executeBinding = [=]() {
+				auto depth = self->ref().depth();
+				if (depth != CV_32F && depth != CV_64F)
+					throw std::runtime_error("dst must have a depth of CV_32F or CV_64F - mat depth:" + std::to_string(depth));
+				cv::accumulateProduct(src1->ref(), src2->ref(), self->ref(), mask->ref());
```

Comment:
Similar here: `cv::accumulateProduct(self->ref(), src2->ref(), dst->ref(), mask->ref());`

Reply:
please see the last review's reply